src/platform/user/profile/utilities/index.js

```diff
@@ -108,9 +112,9 @@ export function mapRawUserDataToState(json) {
     const errorStatus = meta.errors.find(
       error => error.externalService === commonServices.MVI,
     ).status;
-    userState.status = getErrorStatusDesc(errorStatus);
+    userState.veteranStatus.status = getErrorStatusDesc(errorStatus);
   } else {
-    userState.status = vaProfile.status;
+    userState.veteranStatus.status = vaProfile.status;
```

Comment:
`userState.status` should continue to come from `vaProfile.status`. The `status` that ends up on the `veteranStatus` object comes directly from `GET user` API call.

Reply:
Updated! Thanks for being a second pair of eyes on this!